src/content/Effects.js

```diff
@@ -145,13 +145,13 @@ def foreach(f: a ~> Unit, l: List[a]): Unit & Impure`}
 
                 <p>
                     Note that here <Code>ef1</Code> and <Code>ef2</Code> are arbitrary boolean variables
-                    which are not directly associated with the effect of <Code>f</Code> or 
```

Comment:
ah, looks like this issue was always addressed. Should I rollback this space changes?

Reply:
Sorry, what happened?

Reply:
![Screenshot from 2022-01-03 13-55-09](https://user-images.githubusercontent.com/2952361/147932952-af0f82de-63fd-4f73-b6c3-0287d6d6e2fa.png)
Here you can see that `f or g` became `f org`. The same problem with `effects of g(`.
I thought that extra space will fix it, but looks like there already space present.

Reply:
I have this problem on iPhone SE 2. Issue is reproducible using firefox's device simulation view.

Reply:
OK. No idea unfortunately.

Reply:
I think if we make a change so that `<Code>xyz</Code>` is not at the end/beginning of a line, that should fix it.

Reply:
I don't know why `Code` tag is rendered without spaces around. I don't know how to build this project to try to fix it. Should I revert this extra spaces changes, so we can merge all other fixes?

Reply:
OK, let us do that then!